templates/rapidsai-core/partials/patch.dockerfile.j2

```diff
@@ -2,8 +2,7 @@
 
 {# Patch for CVE-2020-8116 https://github.com/advisories/GHSA-ff7x-qrg7-qggm #}
 RUN source activate rapids \
-    && npm i -g npm@">=7" \
-    && rm -rf /tmp/yarn* # Removes cached yarn directories from tmp that have outdated ssri/y18n packages
```

Comment:
Do the new changes handle the yarn dirs in tmp?

Reply:
yup, the cache clean commands remove everything that was vulnerable.